Common/MathUtils/include/MathUtils/Trans3D.h

```diff
@@ -0,0 +1,75 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_BASE_TRANS3D_H_
+#define ALICEO2_BASE_TRANS3D_H_
+
+#include "MathUtils/Cartesian3D.h"
+#include <Math/GenVector/Transform3D.h>
+#include <TGeoMatrix.h>
+#include <Rtypes.h>
+ 
+
+namespace o2 {
+namespace Base {
+
+class Trans3D : public ROOT::Math::Transform3D {
```

Comment:
Transform3D is double, defined as ````typedef Impl::Transform3D<double> Transform3D;```` in math/genvector/inc/Math/GenVector/Transform3D.h. To specify <double> explicitly we would need to derive from  ROOT::Math::impl::Transform3D<double>. I added some comments with explanations.